app/assets/json/translations/locale-en.json

```diff
@@ -1040,8 +1040,12 @@
             "get_help": {
                 "other_groups_may_qualify": "Others, like San Francisco teachers and first responders, may qualify for loans.",
                 "some_people_may_qualify": "Some people may qualify for downpayment assistance.",
-                "title": "Get help",
-                "view_full_list_of_programs": "View the full list of programs"
+                "title": "Help with downpayments",
+                "DALP_program_2021": "The 2021 Downpayment Assistance Loan Program (DALP) will begin accepting applications on February 26, 2021.",
```

Comment:
I think we should have all of the translations strings be lower-case. Can you also run grunt translations and see how it handles having a number in the translation key?